_sass/custom/_card.scss

```diff
@@ -0,0 +1,66 @@
+.card {
```

Comment:
Question for me is, are all the cards are going to be the same style? For example the mentors page, I think it could have different spaces. But I am not following the best practices about css. So I am just wondering if for maintenance would be better to no affect all the pages with cards. 

![image](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/assets/3664747/7d6eda2e-e5ee-4d36-8320-a44c63151397)

Reply:
@dricazenck for maintaining an ever-evolving UI design system, it's best to keep custom styles at a minimum.
I've added the class "card-m"  to these "50% width" cards. So if we increase the padding for these "medium size" cards, I think it should apply to the homepage CTAs as well. Just tested locally with the padding increased and it looks good, so we can change that :)

Reply:
@dricazenck In general, I would avoid adding too custom styles for specific content. Since this would look more balanced with different content where there are more characters per line for example. Or if these CTAs had featured images.

Reply:
Thanks for the clarification!! you 🚀